components/file_preview_modal/file_preview_modal.jsx

```diff
@@ -261,9 +238,9 @@ export default class FilePreviewModal extends React.PureComponent {
     }
 
     handleModalClose = () => {
-        this.props.onModalDismissed();
-        this.setState({scale: Utils.fillArray(ZoomSettings.DEFAULT_SCALE, this.props.fileInfos.length)});
```

Comment:
we dont have to reset the scale since the component will be remounted when opened and the scales will reset